lib/workers/repository/__fixtures__/master-issue_with_8_PR.txt

```diff
@@ -6,6 +6,7 @@ These branches will be created by Renovate only once you click their checkbox be
 
  - [ ] <!-- approve-branch=branchName1 -->pr1
  - [ ] <!-- approve-branch=branchName2 -->pr2
+ - [ ] <!-- approve-all-pending-prs -->⚠ **Approve all pending PRs** ⚠
```

Comment:
Do we want to label this `breaking` and put it in a new `major` release of Renovate?

My thinking is that this is a big change to the amount of "noise/mess" you can make when clicking a checkbox on the Dependency Dashboard. I think self-hosted Renovate users in large organization would like a proper warning about this change.

What do you maintainers think?

Reply:
I don't think we need to consider it major or warn. There are many other ways already to create noise and the user shouldn't do this by accident